webapp/templates/jury/import_export.html.twig

```diff
@@ -6,110 +6,120 @@
-                            </li>
-                        </ul>
-                    </li>
-                </ul>
+            <div class="tile">
+                <h3>Import JSON</h3>
+                {{ form(contest_import_form) }}
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="tile">
+                <h3>Export JSON</h3>
+                {{ form(contest_export_form) }}
             </div>
         </div>
     </div>
 
+
+    <h2 id="teams">Teams &amp; groups</h2>
     <div class="row">
         <div class="col-md-6">
-            <div class="jumbotron">
-                <h4>Tab-separated import</h4>
+            <div class="tile">
+            <h3>Import tab-separated</h3>
                 {{ form(tsv_form) }}
             </div>
         </div>
+        <div class="col-md-6">
+            <div class="tile">
+                <h3>Export tab-separated</h3>
+                <ul>
+                {% for type in ['groups', 'teams'] %}
+                    <li>
+                        <a href="{{ path('jury_tsv_export', {'type': type}) }}"><code>{{ type }}.tsv</code></a>
+                    </li>
+                {% endfor %}
+                </ul>
+            </div>
+        </div>
     </div>
-
     <div class="row">
         <div class="col-md-6">
-            <div class="jumbotron">
-                <h4>JSON import</h4>
+            <div class="tile">
+            <h3>Import JSON</h3>
                 {{ form(json_form) }}
             </div>
         </div>
     </div>
-
     <div class="row">
         <div class="col-md-6">
-            <div class="jumbotron">
-                <h3>Import teams from ICPC CMS</h3>
-                <div class="alert alert-info">
-                    Create a "Web Services Token" with appropriate rights in the "Export" section for your contest at <a
-                            href="https://icpc.global/login" target="_blank">https://icpc.global/login</a>.
-                    You can find the Contest ID (e.g. <code>Southwestern-Europe-2014</code>) in the URL.
-                </div>
-
+            <div class="tile">
+            <h3>Import teams from ICPC CMS</h3>
                 {{ form_start(icpccms_form) }}
                 {{ form_row(icpccms_form.contest_id) }}
                 {{ form_row(icpccms_form.access_token) }}
-                <div class="form-group">
-                    {{ form_widget(icpccms_form.fetch_teams) }}
-                    {{ form_widget(icpccms_form.upload_standings, {'disabled': true, attr: {'title': 'Non-functional because of API changes. See results.tsv above, upload this in the ICPC CMS web interface.'}}) }}
-                </div>
+                {{ form_widget(icpccms_form.fetch_teams) }}
                 {{ form_end(icpccms_form) }}
             </div>
         </div>
     </div>
 
+    <h2 id="results">Results</h2>
+    <div class="row">
+        <div class="col-md-6">
+        </div>
+        <div class="col-md-6">
```

Comment:
`col-md-push-6` on this one instead of the empty `col-md-6` above?

Reply:
Doesn't work for me...

Reply:
😢 apparently it's `col-md-offset-6`: https://getbootstrap.com/docs/3.4/css/#grid-offsetting